staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -606,6 +612,18 @@ func (c completedConfig) New(name string, delegationTarget DelegationTarget) (*G
 		}
 	}
 
+	requestManagerConfigControllerHookName := "apiserver/request-management-config-controller"
+	if feature.DefaultFeatureGate.Enabled(features.APIPriorityAndFairness) && !s.isPostStartHookRegistered(requestManagerConfigControllerHookName) {
+		err := s.AddPostStartHook(requestManagerConfigControllerHookName, func(context PostStartHookContext) error {
+			go c.FlowControl.Run(context.StopCh)
```

Comment:
do we have to run this in the background even if somebody sets `--enable-inflight-quota-handler=false` in the flags? (but w/ the feature-gate enabled)

Reply:
Yes.  `c.FlowControl.Run` runs indefinitely, but the post-start-hook should return quickly.

Reply:
I think the question is, does this have to run *at all* if `--enable-inflight-quota-handler=false`?

Reply:
My first question is, why is there a `--enable-inflight-quota-handler` flag?

Reply:
The name is a bit odd to me, but it came from @yue9944882 in PR #76413; it controls the use of the new priority & fairness code (when it's off, we should be using the previous max-in-flight code).

Reply:
We've renamed the feature and no one should be using the flag yet so it's probably a good time to fix the name.